recipe/meta.yaml

```diff
@@ -1,14 +1,17 @@
 {% set name = "ovito" %}
-{% set version = "3.11.0" %}
-{% set sha256 = "42bd74a86214046acfdb3dd0595103866fd929788684684c544945a9c54b3d53" %}
+{% set version = "3.11.1" %}
+{% set sha256 = "e1be7aa81b6c4bb744fefbc18508542c8553b3b9e70e48f0f467f4de59fe75fe" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://gitlab.com/stuko/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  - url: https://gitlab.com/stuko/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://github.com/facebook/zstd/releases/download/v1.5.6/zstd-1.5.6.tar.gz
```

Comment:
Why do you need this if you are using cf zstd

Reply:
We primarily need it for the header files